<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacroControlWidget</class>
 <widget class="QWidget" name="MacroControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1578</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MacroControlWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="foo_0">
     <property name="title">
      <string>Macro Control</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListView" name="StepView"/>
          </item>
          <item>
           <widget class="QGroupBox" name="foo_1">
            <property name="title">
             <string>Steps</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QLineEdit" name="StepNameLineEdit">
               <property name="inputMask">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Step Name</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="foo_0_0">
               <item>
                <widget class="QPushButton" name="MoveUpBtn">
                 <property name="text">
                  <string>Move Up</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="MoveDownBtn">
                 <property name="text">
                  <string>Move Down</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="AddStepBtn">
                 <property name="text">
                  <string>Add Step</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="DeleteBtn">
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <widget class="QListView" name="ActionView"/>
         <widget class="QListView" name="EndConditionView"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="foo_1_0">
       <property name="title">
        <string>Operations</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTabWidget" name="ActionTab">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="ActionAddBtn">
            <property name="text">
             <string>Add to Step</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ActionSetBtn">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ActionDeleteBtn">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="foo_1_1">
          <property name="title">
           <string>End Condition</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="QTabWidget" name="EndConditionTab">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tab_3">
              <attribute name="title">
               <string>Tab 1</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="tab_4">
              <attribute name="title">
               <string>Tab 2</string>
              </attribute>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="EndConditionAddBtn">
               <property name="text">
                <string>Add to Step</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="EndConditionSetBtn">
               <property name="text">
                <string>Set</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="EndConditionDeleteBtn">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="foo_2">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Sequence Control</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="OpenMacroBtn">
        <property name="text">
         <string>Open Macro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveMacroBtn">
        <property name="text">
         <string>Save Macro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RunMacroBtn">
        <property name="text">
         <string>Run Macro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ForceStopBtn">
        <property name="text">
         <string>Stop Macro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveDataBtn">
        <property name="text">
         <string>Save Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
